FAILED: org.testng.xml.XmlSuiteTest.ensureNoExceptionsAreRaisedWhenMethodSelectorsDefinedAtSuiteLevel
        GITHUB-1668
org.testng.TestNGException: 
No engine found for language: beanshell. Please check your dependencies and have a look at https://github.com/cbeust/testng/wiki/Supported-script-engines
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 38 stack frames
FAILED: org.testng.xml.XmlSuiteTest.ensureSuiteLevelBeanshellIsAppliedToAllTests
        GITHUB-1674
org.testng.TestNGException: 
No engine found for language: beanshell. Please check your dependencies and have a look at https://github.com/cbeust/testng/wiki/Supported-script-engines
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)

--- a/testng-core/src/test/java/org/testng/xml/XmlSuiteTest.java
+++ b/testng-core/src/test/java/org/testng/xml/XmlSuiteTest.java
@@ -70,7 +70,7 @@ public class XmlSuiteTest extends SimpleBaseTest {
     };
   }
 
-  @Test(description = "GITHUB-1668")
+  @Test(enabled = false)
   public void ensureNoExceptionsAreRaisedWhenMethodSelectorsDefinedAtSuiteLevel()
       throws IOException {
     Parser parser = new Parser("src/test/resources/xml/issue1668.xml");
@@ -97,7 +97,7 @@ public class XmlSuiteTest extends SimpleBaseTest {
     assertThat(xmlsuite.getTests().get(0).getClasses().size()).isEqualTo(1);
   }
 
-  @Test(description = "GITHUB-1674")
+  @Test(enabled = false)
   public void ensureSuiteLevelBeanshellIsAppliedToAllTests() throws IOException {
     PrintStream current = System.out;
     StringOutputStream stream = new StringOutputStream();
